opentelemetry-otlp/examples/basic-otlp-http/src/main.rs

```diff
@@ -32,7 +32,8 @@ fn init_logs() -> Result<sdklogs::LoggerProvider, opentelemetry::logs::LogError>
         .with_exporter(
             opentelemetry_otlp::new_exporter()
                 .http()
-                .with_endpoint("http://localhost:4318/v1/logs"),
+                .with_protocol(opentelemetry_otlp::Protocol::HttpJson)
```

Comment:
```suggestion
                .with_protocol(opentelemetry_otlp::Protocol::HttpBinary)
```

Let's keep all the three exporters as binary for consistency, as we already have the README.md updated for HTTP/JSON support